kafka-ui-react-app/src/components/Wizard/WizardForm/Metrics/Metrics.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import * as S from 'components/Wizard/WizardForm/WizardForm.styled';
+import { Button } from 'components/common/Button/Button';
+import Input from 'components/common/Input/Input';
+import { useFormContext } from 'react-hook-form';
+import Heading from 'components/common/heading/Heading.styled';
+import ControlledSelect from 'components/common/Select/ControlledSelect';
+import { METRICS_OPTIONS } from 'lib/constants';
+import Checkbox from 'components/common/Checkbox/Checkbox';
+
+const Metrics = () => {
+  const [newMetrics, setNewMetrics] = useState(false);
+  const { reset, getValues, watch } = useFormContext();
+  const showMetrics: React.MouseEventHandler<HTMLButtonElement> = (e) => {
+    e.preventDefault();
+    setNewMetrics(!newMetrics);
+    if (!newMetrics) {
+      reset(
+        {
+          ...getValues(),
+          Metrics: {
```

Comment:
uppercase?

Reply:
ok i changed, it's because in a schema it wrote uppercase